src/main/java/com/ericsson/ei/controller/model/ParseInstanceInfoEI.java

```diff
@@ -37,21 +36,26 @@
  */
 @Component
 public class ParseInstanceInfoEI {
+	
     @Getter
     @Value("${spring.application.name}")
     private String applicationName;
 
     @Getter
     @Value("${build.version}")
     private String version;
+    
+    @Getter
+    @Value("${testaggregated.enabled}")
```

Comment:
you can set  default value for this property here (like ("${testaggregated.enabled:false}").  then even if this property is missed in application.properties, you will not get error.